Data_Examples/README.md

```diff
@@ -0,0 +1,6 @@
+This folder is meant to be used for examples of datasets from each consortium which are annotated using common 
+terminologies. The goal of such examples is the testing of frameworks such as CDIF or I-ADOPT to have some practical 
```

Comment:
Maybe add a links to CDIF book `https://cross-domain-interoperability-framework.github.io/cdifbook/` and i-adopt `https://i-adopt.github.io/` here?

Reply:
Thanks you, good point! Will add this.